Google.PowerShell/Common/ReportCmdletResults.cs

```diff
@@ -174,32 +180,33 @@ public GoogleAnalyticsCmdletReporter(string clientID, AnalyticsEventCategory ana
             }
         }
 
-        public void ReportSuccess(string cmdletName, string parameterSet)
+        public void ReportSuccess(string cmdletName, string parameterSet, string projectNumber)
         {
-            Report(cmdletName, parameterSet, null);
+            Report(cmdletName, parameterSet, projectNumber, null);
         }
 
-        public void ReportFailure(string cmdletName, string parameterSet, int errorCode)
+        public void ReportFailure(string cmdletName, string parameterSet, string projectNumber, int errorCode)
         {
-            Report(cmdletName, parameterSet, errorCode);
+            Report(cmdletName, parameterSet, projectNumber, errorCode);
         }
 
-        private void Report(string cmdletName, string parameterSet, int? errorCode)
+        private void Report(string cmdletName, string parameterSet, string projectNumber, int? errorCode)
         {
             // The event type will be CmdletInvocation or ProviderInvocation. The specific cmdlet and parameter set
             // used are encoded in the event's metadata.
             AnalyticsEvent eventData = new AnalyticsEvent(
                 _category.ToString(),
                 "cmdletName", cmdletName,
                 "parameterSet", (parameterSet ?? "null"),
+                "projectNumber", (projectNumber ?? "null"),
                 "errorCode", (errorCode.HasValue ? errorCode.ToString() : "null"));
 
             s_reporter.Value.ReportEvent(
                 source: "virtual.powershell",
                 eventType: "powershell",
                 eventName: eventData.Name,
                 userLoggedIn: true,
-                projectNumber: null,
+                projectNumber: projectNumber,
```

Comment:
@ivannaranjo just to make sure I have that right, _hash of the project number_? e.g. `"96463850128".GetHashCode()`?

I assume if we don't do that exactly, and instead just hash the project name, that will be OK. Right? Since on our backend logs the entire reason we hash things is so that it is anonymized, and if the anonymization method we use isn't compatible, that only breaks cases where you are joining different data streams. Something that shouldn't be done anyways, right?

@quoctruong As long as the code is consistent (variable names and comments) around projectName vs. projectNumber vs. projectXXHash, I'm happy with this PR.